exercises/4_1/exercise_10.js

```diff
@@ -0,0 +1,11 @@
+let costPrice = 100;
+let sellPrice = 200;
+let profit = 0;
+
+if (costPrice > 0 || sellPrice > 0) {
```

Comment:
Olá Daniel,
Será que não seria melhor trocar o operador "ou" `||` por um "e" `&& ` ? Do jeito que está, se apenas um dos valores for igual ou menor que zero a mensagem de erro não aparece.

Reply:
Olá Lucas,
Muito obrigado pelo Review :)
Fui reler o enunciado: "Seu programa também deve emitir uma mensagem de erro e encerrar caso algum dos seus valores de entrada seja menor que zero."
Eu acho que é melhor o operador "ou" mesmo nesta situação, já que é para emitir mensagem de erro caso ALGUM DOS VALORES seja menor que zero. Ou seja, se um OU outro for menor que zero já daria ERRO. 
No caso do operador "e" para dar ERRO ambos devem ser abaixo de zero. Acredito que o enunciado do exercício seria mais ou menos assim "...caso AMBOS os valores sejam menor que zero..."

Reply:
Ooops... Acho que eu inverti aqui.